packages/js-plugin-aws/src/plugin.ts

```diff
@@ -1,12 +1,15 @@
 import type { S3Client } from '@aws-sdk/client-s3';
-import type { Metaplex, MetaplexPlugin } from '@metaplex-foundation/js';
+import type {
+  Metaplex as MetaplexType,
+  MetaplexPlugin,
+} from '@metaplex-foundation/js';
 import { AwsStorageDriver } from './AwsStorageDriver';
 
 export const awsStorage = (
   client: S3Client,
   bucketName: string
 ): MetaplexPlugin => ({
-  install(metaplex: Metaplex) {
+  install(metaplex: MetaplexType) {
```

Comment:
Why? Is the new lint doing this?

Reply:
yes Metaplex is defined twice, one is shadowed by other. I renamed type as a type

Reply:
How is it being defined twice? I can see only one `Metaplex` instance.

https://github.com/metaplex-foundation/js/blob/f3c86ea965d5dd05ed6598c491764c1fe4194a81/packages/js-plugin-aws/src/plugin.ts

Reply:
disabled. reverted changes related to Metaplex type